Services/CO.CDP.Organisation.WebApi.Tests/UseCase/InvitePersonToOrganisationUseCaseTest.cs

```diff
@@ -0,0 +1,94 @@
+using CO.CDP.GovUKNotify;
+using CO.CDP.Organisation.WebApi.Model;
+using CO.CDP.Organisation.WebApi.UseCase;
+using CO.CDP.OrganisationInformation.Persistence;
+using GovukNotify.Models;
+using Microsoft.Extensions.Configuration;
+using Moq;
+using Persistence = CO.CDP.OrganisationInformation.Persistence;
+
+namespace CO.CDP.Organisation.WebApi.Tests.UseCase;
+
+public class InvitePersonToOrganisationUseCaseTest
+{
+    private readonly Mock<Persistence.IOrganisationRepository> _organisationRepository = new();
+    private readonly Mock<Persistence.IPersonInviteRepository> _personsInviteRepository = new();
+    private readonly Mock<IGovUKNotifyApiClient> _mockGovUKNotifyApiClient = new();
+    private readonly IConfiguration _mockConfiguration;    
+    private readonly Guid _generatedGuid = Guid.NewGuid();
+
+    private InvitePersonToOrganisationUseCase _useCase => new(
+        _organisationRepository.Object,
+        _personsInviteRepository.Object,
+        _mockGovUKNotifyApiClient.Object,
+        _mockConfiguration,
+        () => _generatedGuid);
+
+    public InvitePersonToOrganisationUseCaseTest()
+    {
+        var inMemorySettings = new Dictionary<string, string> {
+            {"GOVUKNotify:PersonInviteEmailTemplateId", "test-template-id"}
+        };
+
+        _mockConfiguration = new ConfigurationBuilder()
+            .AddInMemoryCollection(inMemorySettings)
+            .Build();
+    }
+
+    [Fact]
+    public async Task Execute_ValidOrganisationAndInvite_SuccessfullySendsInvite()
+    {
+        var organisationId = Guid.NewGuid();
+        InvitePersonToOrganisation invitePersonData = CreateDummyInviteToPerson();
+
+        var organisation = new Persistence.Organisation
+        {
+            Guid = organisationId,
+            Name = "Test Organisation",
+            Tenant = It.IsAny<Tenant>()
+        };
+
+        var command = (organisationId, invitePersonData);
+
+        _organisationRepository.Setup(repo => repo.Find(organisationId))
+            .ReturnsAsync(organisation);
+
+        _mockGovUKNotifyApiClient.Setup(client => client.SendEmail(It.IsAny<EmailNotificationResquest>()));
+
+        var result = await _useCase.Execute(command);
+
+        Assert.NotNull(result);
```

Comment:
I was told to avoid using Assert by Kuba previously https://goacoglobal.slack.com/archives/C06M34UGL59/p1725900922624619?thread_ts=1725895488.975309&cid=C06M34UGL59

Reply:
Indeed, that's just for consistency. The team has agreed to use fluentassertions, so we should use it consistently!